static/app/components/stories/themeSwitcher.tsx

```diff
@@ -14,8 +15,12 @@ export default function ThemeSwitcher() {
   }, [isDark]);
 
   return (
-    <Button size="xs" onClick={handleClick}>
-      {isDark ? t('Switch to Light Mode') : t('Switch to Dark Mode')}
-    </Button>
+    <Button
```

Comment:
That big 'switch to dark mode' button with a label was bothering me, as it is quite large and somewhat visually intrusive. I've swapped it for an icon button